cmd/operator-ui/go.mod

```diff
@@ -40,17 +39,18 @@ require (
 	github.com/gobuffalo/mw-i18n v0.0.0-20190224203426-337de00e4c33
 	github.com/gobuffalo/mw-paramlogger v0.0.0-20190224201358-0d45762ab655
 	github.com/gobuffalo/packr/v2 v2.5.1
+	github.com/gobuffalo/suite v2.8.1+incompatible
```

Comment:
Please add the new deps to the replace section

Reply:
This is automatically added via `make buffalo`.  Why are putting the same dep info in both `replace` and `require` block?

Reply:
Done, but I think we should redo `go.mod` files in both main and operator-ui and keep them consistent across master and release branches, probably from 2019.8.0 onwards.